OpenRA.Game/Activities/Activity.cs

```diff
@@ -162,7 +146,7 @@ public Activity()
 
 		public Activity TickOuter(Actor self)
 		{
-			if (State >= ActivityState.Done && Game.Settings.Debug.StrictActivityChecking)
+			if (State == ActivityState.Done && Game.Settings.Debug.StrictActivityChecking)
```

Comment:
This is reverting an outdated change in the last commit - can you move this into that one so it doesn't show up in the history at all?

Reply:
Fixed.